core/src/main/java/io/questdb/cairo/sql/async/PageFrameReduceTask.java

```diff
@@ -184,6 +184,7 @@ public void resetCapacities() {
     }
 
     public void setErrorMsg(Throwable th) {
+        th.printStackTrace();
```

Comment:
This looks like debug leftover.

Reply:
removed in https://github.com/questdb/questdb/pull/4632/commits/a27f01594461930d022aac04163dce85937301b9